moduleroot/scripts/test/large_spec.rb

```diff
@@ -0,0 +1,113 @@
+require_relative './spec_helper'
+require 'specinfra/backend/docker_compose'
+
+compose_yml = File.expand_path(File.join(__FILE__, "../docker-compose.yml"))
+set :docker_compose_container, :snap
+
+describe docker_compose(compose_yml) do
+  its_container(:snap) do
+    describe 'docker-compose.yml run' do
+      TIMEOUT = 60
+
+      describe "download Snap" do
+        it {
+          expect(cmd_with_retry("/opt/snap/bin/snaptel --version", :timeout => TIMEOUT).exit_status).to eq 0
+          expect(cmd_with_retry("/opt/snap/sbin/snapteld --version", :timeout => TIMEOUT).exit_status).to eq 0
+        }
+      end
+
+      if os[:family] == 'alpine'
```

Comment:
On ubuntu the port detection relies on ss command. If we install the package, for whatever reason in Docker the output is blank:
```
# ss
Netid State      Recv-Q Send-Q Local Address:Port                 Peer Address:Port
```

If we run the container with --host=net, ss outputs information about docker, but we still don't see the port listed, so for now I'm only checking ports on systems using the netstat command.